tests/e2e/support/commands.js

```diff
@@ -16,53 +16,55 @@ Cypress.Commands.add('skipOnGameStateApi', () => {
   }
 });
 
-const transformGameUrl = (api, slug) => {
+const transformGameUrl = (api, slug, gameId = null) => {
   if (!env) {
     return Cypress.Promise.resolve(`/api/${api}/${slug}`);
   }
 
-  if (slug === 'rematch') {
-    return cy
-      .window()
-      .its('cuttle.gameStore.id')
-      .then((gameId) => `/api/game/${gameId}/rematch`);
+  switch (slug) {
```

Comment:
Refactored to `switch()` to more consistently handle different values of the slug